### PR TITLE
23.08 Release preparation

### PR DESCRIPTION
**Pull request by ktangsali**
This PR updates a the base docker image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Dependencies
 
+- Updated the base container to latest PyTorch base container which is based on torch 2.0
```
Comment (akshaysubr) [inline CHANGELOG.md:27]: Shouldn't these be in the `Changed` section?

**Review by akshaysubr — APPROVED** [review]
Approved, just left a minor comment about the changelog

File: Dockerfile
Comment (mnabian) [inline Dockerfile:1]: Shouldn't the dependencies with a minimum version be enclosed within quotes?

Comment (mnabian) [inline Dockerfile:1]: We should also update the Modulus version. That can be a separate MR though

**Review by mnabian — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ktangsali) [reply CHANGELOG.md:27]: Good point Akshay! To be consistent let's leave it the way it is right now and I will make it a point to revisit this in the next set of PRs.

File: Dockerfile
Comment (ktangsali) [reply Dockerfile:1]: Good catch Mohammad, updated it :)

Comment (ktangsali) [reply Dockerfile:1]: Yes, we don't have a pip release of core yet. Once that happens, we can update it.

**Review by mnabian — APPROVED** [review]
LGTM
